src/gengowatcher/watcher.py

```
@@ -210,8 +265,64 @@
+        try:
+            job_data = {
+                "id": str(job_id),
+                "title": title,
+                "reward": float(reward),
+                "currency": "USD",
+                "url": url,
+                "timestamp": time.time(),
+                "source": source
+            }
+            self.state.add_job(job_data)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)